all-clouds/all-clouds-explanation/cloud-init-metapackages.rst

```diff
@@ -0,0 +1,21 @@
+Cloud-init metapackages
+=======================
+
+Starting on Ubuntu Oracular (24.10), `cloud-init <https://launchpad.net/ubuntu/+source/cloud-init>`
+added cloud-specific metapackages to better express cloud-specific dependencies and reduce image size.
+
+* ``cloud-init-base`` contains the cloud-init code with the minimal set of dependencies
+
+* ``cloud-init`` is a metapackage that retains every previous dependency
+
+* ``cloud-init-cloud-sigma`` and ``cloud-init-smart-os`` are metapackages with an
+  additional dependency on python3-serial
+
+If you are an image builder:
+
+* Install ``cloud-init``, containing all dependencies, if the target platform is unknown
+
+* Install a cloud-specific cloud-init package if the target platform requires one
```

Comment:
it took me a second to understand what was being conveyed here. i didn't make the connection between l11 (defining the available cloud specific packages) and this instruction. is there a better way to highlight that in the previous block to make this translate a little easier?

e.g. `... are cloud specific metapackages ...` may be enough of a tweak to make this more consumable

Reply:
Done, thanks.